tests/test_treeherder_menu.py

```diff
@@ -9,15 +9,16 @@
 
 @pytest.mark.nondestructive
 def test_treeherder_dropdown(base_url, selenium):
-    # Switch between Treeherder and Perfherder using header dropdown
+    """ Switch between Treeherder and Perfherder using header dropdown """
     treeherder_page = TreeherderPage(base_url, selenium).open()
-    # Verify Treeherder loads
+
+    """ Verify Treeherder loads """
```

Comment:
Nit: The `"""` style is for docstrings, and should only be used in the first line of a method.
For other comments it's fine to use `#`.